production/helm/loki/values.yaml

```diff
@@ -579,7 +579,7 @@ monitoring:
   # ServiceMonitor configuration
   serviceMonitor:
     # -- If enabled, ServiceMonitor resources for Prometheus Operator are created
-    enabled: true
+    enabled: false
```

Comment:
again here, self monitoring enabled should be the default.

Reply:
From my point of view, its common practice is disable custom resources by default. Helm charts should not depend agaist custom resources unless the CRDs are managed by the helm chart.

The [grafana ](https://github.com/grafana/helm-charts/blob/2cf626aad06a12506c014aa541f85b8646f0916c/charts/grafana/values.yaml#L173)and [tempo](https://github.com/grafana/helm-charts/blob/2cf626aad06a12506c014aa541f85b8646f0916c/charts/tempo/values.yaml#L199) chart are follow this principle, too.

I can revert both mentioned changes to get an non working configuration by default. This is fine for me.

---

But include foreign CRDs feel really hackly, since helm CRD behavior is unexpectable.

Reply:
Please take a look at fe5dfbf, if this fits for you.